src/console/ConsolePage.test.jsx

```diff
@@ -254,15 +261,125 @@ describe('ConsolePage', () => {
       data={[]}
       downloadEnrollments={() => {}}
       fetchPrograms={mock}
+      filterPrograms={() => {}}
       programBanners={{}}
       uploadEnrollments={() => {}}
       removeBanner={() => {}}
-      switchPage={jest.fn()}
+      switchPage={() => {}}
     />);
 
     expect(mock).toHaveBeenCalled();
   });
 
+  it('calls the filterPrograms function on form submit', () => {
+    const mock = jest.fn();
+
+    expect(mock).not.toHaveBeenCalled();
+
+    const apiData = [{
```

Comment:
Seems like this `const` is defined to be the same for all your tests, can we refactor?